analyzer/uncategorized/evm_raw_tx_test.go

```diff
@@ -0,0 +1,229 @@
+package common
+
+import (
+	"encoding/hex"
+	"math/big"
+	"strings"
+	"testing"
+
+	"github.com/oasisprotocol/oasis-core/go/common/cbor"
+	"github.com/oasisprotocol/oasis-core/go/common/quantity"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/helpers"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/modules/evm"
+	sdkTypes "github.com/oasisprotocol/oasis-sdk/client-sdk/go/types"
+	"github.com/stretchr/testify/require"
+)
+
+// These tests are based on oasis-sdk/runtime-sdk/modules/evm/src/raw_tx.rs.
+
+func decodeExpectCall(
+	t *testing.T,
+	raw string,
+	expectedChainID uint64,
```

Comment:
The high number of args makes tests hard to read. Can we introduce a `type Call struct {...}`, then having `expected Call` here and `Call { ChainId: ..., To:, ..., ... }` in tests?

Same for `decodeExpectCreate`.

Reply:
ya that'd be good. let's do that in #158